backend/runprocessors.py

```diff
@@ -126,9 +130,15 @@ def run_scrape_enrolment_data():
         "program": cache_program_mappings,
         "graph": cache_graph
     },
+    "program_condition": {
```

Comment:
ummm doesnt this cause `python runprocessors` to raise an error guaranteed? You should add something to avoid calling these until they are implemented.

Reply:
Just commented out the `process` key because that one's still WIP and not done. The other two should be good to go